curriculum/challenges/english/02-javascript-algorithms-and-data-structures/basic-javascript/testing-objects-for-properties.english.md

```diff
@@ -32,13 +32,16 @@ Modify the function <code>checkObj</code> to test <code>myObj</code> for <code>c
 
 ```yml
 tests:
-  - text: <code>checkObj("gift")</code> should return  <code>"pony"</code>.
-    testString: assert(checkObj("gift") === "pony");
-  - text: <code>checkObj("pet")</code> should return  <code>"kitten"</code>.
-    testString: assert(checkObj("pet") === "kitten");
-  - text: <code>checkObj("house")</code> should return  <code>"Not Found"</code>.
-    testString: assert(checkObj("house") === "Not Found");
-
+  - text: '<code>checkObj({gift: "pony", pet: "kitten", bed: "sleigh"}, "gift")</code> should return  <code>"pony"</code>.'
+    testString: 'assert(checkObj({gift: "pony", pet: "kitten", bed: "sleigh"}, "gift") === "pony");'
+  - text: '<code>checkObj({gift: "pony", pet: "kitten", bed: "sleigh"}, "pet")</code> should return  <code>"kitten"</code>.'
+    testString: 'assert(checkObj({gift: "pony", pet: "kitten", bed: "sleigh"}, "pet") === "kitten");'
+  - text: '<code>checkObj({gift: "pony", pet: "kitten", bed: "sleigh"}, "house")</code> should return  <code>"Not Found"</code>.'
+    testString: 'assert(checkObj({gift: "pony", pet: "kitten", bed: "sleigh"}, "house") === "Not Found");'
+  - text: '<code>checkObj({"city": "Seattle"}, "city")</code> should return  <code>"Seattle"</code>.'
+    testString: 'assert(checkObj({"city": "Seattle"}, "city") === "Seattle");'
+  - text: '<code>checkObj({"city": "Seattle"}, "district")</code> should return  <code>"Not Found"</code>.'
```

Comment:
```suggestion
  - text: '<code>checkObj({city: "Seattle"}, "district")</code> should return <code>"Not Found"</code>.'
```

Reply:
Removed all the double spaces from return, removed double quotes in in object key